src/frontend/magnify/src/components/design-system/RowsList/RowsList.tsx

```diff
@@ -155,14 +161,16 @@ export default function RowsList<TRowProps extends Row>({
         </Box>
       </Grid>
       {Header && <Header selected={selected} setSelected={setSelected} />}
-      {rows.map((row) => (
-        <Row
-          key={row.id}
-          {...row}
-          selected={selected[row.id]}
-          onToggle={() => handleToggle(row.id)}
-        />
-      ))}
+      {!isLoading &&
+        rows.map((row) => (
+          <Row
+            key={row.id}
+            {...row}
+            selected={selected[row.id]}
+            onToggle={() => handleToggle(row.id)}
+          />
+        ))}
+      {isLoading && [1, 2, 3].map((v) => <WaitingRow key={v} />)}
```

Comment:
What is `[1, 2, 3]` ? Is it for demo purpose ?

Reply:
IMO, if it's only for demo, I'll remove the map totally:
```suggestion
      {isLoading && (
          <>
              <WaitingRow />
              <WaitingRow />
              <WaitingRow />
          </>
      )}
```

Reply:
It's not for demo. It's the 3 blinking rows replacing the other rows before they are loaded (see the second screenshot)

Reply:
(We can't know before the request finish how many rows it will be so we need to make an arbitrary choice here)